src/uct/ib/base/ib_md.c

```diff
@@ -501,7 +502,7 @@ static ucs_status_t uct_ib_md_reg_mr(uct_ib_md_t *md, void *address,
                 memh->flags |= UCT_IB_MEM_MULTITHREADED;
             } else {
                 uct_ib_md_print_mem_reg_err_msg(level, address, length,
-                                                access_flags);
+                                                access_flags, errno);
```

Comment:
are we sure no function overwrites "errno" inside reg_multithreaded/reg_key?

Reply:
at least for now - there is no errno overwrite.
it may be changed, but to eliminate this we have to change external API